### DISCUSSION TITLE
Github Workflow for Bchecks Validation

### DISCUSSION THREAD

**Discussion by genuinemoses** [PortSwigger/BChecks#17] [Ideas]
Hi guys,

Do you think it's better to have a Github Actions Workflow when a PR is made to merge a Bcheck?

The main objective of the workflow should be to validate a Bcheck before merging a PR on the fly.

**Comment by genuinemoses** [comment]
It looks like we already have one, haven't noticed it in the workflows folder.

**Comment by genuinemoses** [comment]
@A-J-C, I feel that BChecksValidator is a better name than BChecksChecker for 'BChecksChecker.jar' file. 
Let me know your thoughts.

**Comment by A-J-C** [comment]
Ah sorry @genuinemoses completely missed this conversation. Yes we introduced a validation step as a workflow, as well as a few internal notifications around PR and issue creation. 

If there are any other ideas of flows you wanted to created we can definitely discuss! 
And the name is just a little bit of fun, if you want it changed though feel free to create a PR.

**Reply by genuinemoses** [reply]
No worries, @A-J-C! And, thanks for considering the request.
I'll think about the other workflows that could be purposeful.
And, will soon create a PR with appropriate changes.
